### PR TITLE
Add more emoticons and fix Ryver markdown formatting issue.

### LINKED ISSUE #73: Send emoticons in code formatting
For the `!emoticon` command, all emoticons should be sent in an inline code block (wrapped with backticks) to preserve any characters that have meaning in Markdown.

This also means that any emoticons with backticks should be removed, at least until we come up with a better way to do this.

### PR DESCRIPTION
**Pull request by LukeG294**
Closes #73 

## Summary

Added more emoticons, fixed formatting issue.

## Checklist
- [x] I have read and agree to follow the contributing guidelines and code of conduct in `CONTRIBUTING.md`.
- [x] An issue was made and assigned to me by a maintainer before opening this pull request.
- [x] I have formatted the code as detailed in `CONTRIBUTING.md`.
- [x] I have tested all the code in my pull request, including by running BrainBot as a whole.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -222,14 +222,17 @@ async def _on_chat(msg):
...
                     emoticons = [
-                        "( ͡❛ ͜ʖ ͡❛)",
-                        "O_o",
-                        "（　´_ゝ`）",
-                        "(╯°□°）╯︵ ┻━┻",
-                        ":-)",
-                        "<(o_o<)",
-                        "(/^▽^)/",
-                        "(•‿•)",
+                        "`( ͡❛ ͜ʖ ͡❛)"`,
```
Comment (bsoyka) [inline main.py:225]:
```suggestion
                        "`( ͡❛ ͜ʖ ͡❛)`",
```

File: main.py [hunk 2]
```diff
@@ -222,14 +222,17 @@ async def _on_chat(msg):
...
+                        "`O_o`",
+                        "`（　´_ゝ`）`",
```
Comment (bsoyka) [inline main.py:227]: Take this one out as it contains a backtick itself and won't show up right.

File: main.py [hunk 3]
```diff
@@ -222,14 +222,17 @@ async def _on_chat(msg):
...
+                        "`(╯°□°）╯︵ ┻━┻`",
+                        "`:-)`",
+                        "`<(o_o<)`",
+                        "`(/^▽^)/`",
+                        "`(•‿•)`",
+                        "`〠_〠",
```
Comment (bsoyka) [inline main.py:233]:
```suggestion
                        "`〠_〠`",
```

**Review by bsoyka — DISMISSED** [review]

File: main.py (hunk 1 above)
Comment (LukeG294) [reply main.py:225]: Whoops, thanks,

File: main.py (hunk 2 above)
Comment (bsoyka) [reply main.py:227]: (@LukeG294)

Comment (LukeG294) [reply main.py:227]: Missed that comment originally, thanks.

Comment (bsoyka) [reply main.py:227]: No worries. Also, looks like you just removed the wrong one. :joy:

Comment (LukeG294) [reply main.py:227]: 🤦

Comment (LukeG294) [reply main.py:227]: (•‿•) needs to go, right?

Comment (LukeG294) [reply main.py:227]: Or is it ( ´_ゝ` ) ?

Comment (LukeG294) [reply main.py]: I *think* it should be fixed now.

**Review by bsoyka — APPROVED** [review]

File: main.py (hunk 2 above)
Comment (LukeG294) [reply main.py]: (I just changed it instead of removing it altogether)

**Pull request merged**